### ISSUE TITLE
Unable to log in to user account

### ISSUE THREAD

**Issue by jmtiong** [unee-t/frontend#872]
https://snippet.uilicious.com/embed/test/private/Bzf9tAzuMpK3yizY6Da8F9?step=1&autoplay=1
At step 65 onwards, the test failed due to the system unable to log in to the user account.

When I manually did a manual login, it could not do so as well. Looking into the console tabs, there are errors from the API indicating that the user is not found.
![image](https://user-images.githubusercontent.com/16659123/64143721-df89ee00-ce43-11e9-9ea1-fc899f6f0c02.png)

When I attempt to sign up this user, there is an API error as well.
![image](https://user-images.githubusercontent.com/16659123/64143733-ea448300-ce43-11e9-87c2-f74a3c27c1f5.png)

**Comment by jmtiong** [comment]
Looking into the details of the error message
```
"Error: [[object Object]]
    at w._livedata_result (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:63:17834)
    at w.onMessage (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:63:20175)
    at https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:2772
    at Array.forEach (<anonymous>)
    at l.forEachCallback (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:3967)
    at k.socket.onmessage (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:2742)
    at k.r.dispatchEvent (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:6474)
    at k._dispatchMessage (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:20111)
    at k._didMessage (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:21011)
    at WebSocket.o.ws.onmessage (https://case.demo.unee-t.com/bc718099c4bfbe691c7508c34c88c18a1ebb0fd4.js?meteor_js_resource=true:61:23160)"
```

This is what I obtained during the sign up
![image](https://user-images.githubusercontent.com/16659123/64143796-2aa40100-ce44-11e9-961d-d2686e757b23.png)

**Comment by kaihendry** [comment]
This test seems to assume a **uneet-static-test@inboxkitten.com** account exists, to be logged into.

I am not sure how you can make that assumption?!

**Comment by jmtiong** [comment]
This particular account was created for this testing purpose, however, even so when we try to create this non-existence user, the sign up failed as well.
![image](https://user-images.githubusercontent.com/16659123/64143968-072d8600-ce45-11e9-932b-8fe9378320ef.png)

So if neither the log in works nor the sign up works for this particular account, there should be a problem within the API server?

**Comment by franck-boullier** [comment]
I just did the following test:
- In the DEMO environment
- Create a new user - PASSED
- check the email confirmation is received - PASSED
- click on the link in email confirmation email. Expected result: I'm logged in to Unee-T - PASSED
- Log out PASSED
- Log back in with the user credentials I just created - PASSED

AFAICT the DEMO envo works as expected. I see no API issue or anything out of the ordinary.
